runner/src/main.rs

```diff
@@ -65,6 +65,9 @@ struct BuildArgs {
     /// Build a firmware instead of Miralis
     #[arg(short, long)]
     firmware: Option<String>,
+    #[arg(long)]
+    /// Build miralis as firmware
+    as_firmware: Option<bool>,
```

Comment:
We might not need this extra flag if `miralis` can be passed as the firmware name.

Reply:
Agree, se previous comment